src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2404,7 +2416,7 @@ void processContextElement
     /* All the attributes existing in the request are added to the response with 'found' set to false
      * in the of UPDATE/DELETE and true in the case of APPEND
      */
-    bool foundValue = (strcasecmp(action.c_str(), "append") == 0);
+    bool foundValue = (strcasecmp(action.c_str(), "append") == 0) || (strcasecmp(action.c_str(), "appendonly") == 0);
```

Comment:
Or expand that v1 functionality for the same price to make happy all our existing users that have requested it :).

In the later case, we should mention it in the CNR as a v1 feature and also modify .md documentation to explain the new funcionality (I can do the .md part).